### ISSUE TITLE
Graph and prose aren't in agreement

### ISSUE THREAD

**Issue by jduckles** [carpentries/assessment-projects#18]
You say 53% of respondents are in the US, but the graph shows less than 50%
![image](https://user-images.githubusercontent.com/119403/30006968-a9478368-9158-11e7-866f-f6924aa7ecd6.png)

**Comment by kariljordan** [comment]
@jduckles Yikes! That's such a great catch. Thank you. I am trying to figure out why it's not calculating the correct percentage, because the others are correct. This is the code:

```
data_country_tally <-
data %>%
  group_by(Country) %>%
  tally() %>%
  mutate(perc = round(100 * (n/sum(n)), 1)) %>% # add the % col
  filter(!is.na(Country)) %>%
  arrange(desc(n))

ggplot(data_country_tally,
       aes(reorder(Country, perc),
           perc)) +
  geom_bar(stat = "identity", fill = "orange") +
  theme_classic() +
  xlab("") +
  ylab("Percentage of all participants") +
  coord_flip() +
  ggtitle("Carpentry Workshop Respondents by Country") +
  theme(plot.title = element_text(hjust = 0.5)) +
  theme_bw(base_size = 14)

#Use code below for in-line text.
USA <- data_country_tally[data_country_tally$Country == 'USA', ]$n
Canada <- data_country_tally[data_country_tally$Country == 'Canada', ]$n
UK <- data_country_tally[data_country_tally$Country == 'UK', ]$n
Australia <- data_country_tally[data_country_tally$Country == 'Australia', ]$n
```
Then I use this to pull the numbers in-line with the text:
```
A large portion of Carpentry learners responding to the survey attended a workshop in the United States (`r Country[22]`%), followed by Canada (`r Country[3]`%) the UK (`r Country[21]`%), and Australia (`r Country[1]`%).
```
Does anything stand out to you @benmarwick? I've gone through the code over and over, but can't figure out why it's pulling incorrect numbers.

**Comment by benmarwick** [comment]
Yes, the problem is that the graph and the values that appear in the text do not use the same data!

The graph uses a data frame where the NA rows have been removed.

The inline code that generates the number in the text uses a different data frame that includes the NA rows in the denominator when calculating the proportions. 

So the values in the text are wrong, and the values in the graph are correct. 

I've made the correction here, so that the values in the text use the same data frame as the values in the plot: https://github.com/carpentries/assessment-projects/pull/20, let me know what you think!
